cmd/stat.go

```diff
@@ -127,6 +127,7 @@ Likewise, it produces a similar file that also shows nested numbers, with these
     12 = compresed (.bzip2 | .gz | .tgz | .zip | .xz | .bgz suffix)
     13 = text (.csv | .tsv | .txt | .text | .md | .dat | readme suffix)
     14 = log (.log | .out | .o | .err | .e | .err | .oe suffix)
+    15 = directory
```

Comment:
Does directory type actually appear in output files, and is it a bug if it does?

Reply:
It's doesn't appear in output files; not sure why it got added to the docs. Removed.